lib/services/devices/devices-NGSI-LD.js

```diff
@@ -76,7 +76,7 @@ function createInitialEntityHandlerNgsiLD(deviceData, newDevice, callback) {
             alarms.raise(constants.ORION_ALARM, error);
 
             callback(error);
-        } else if (response && response.statusCode === 204) {
+        } else if (response && (response.statusCode === 200 || response.statusCode == 201 || response.statusCode == 204)) {
```

Comment:
Maybe a comment above this line should explain which spec version uses each "flavour".

Reply:
I tend not to overload the code when the info is easily findable in Git (I've put all the details in the commit message). But as you wish 😄

Reply:
Not everybody is used to check commit messages to look for information ;), so I think a similar comment in the code doesn't hurt and could help, pls.

Reply:
Added some comments.